working-groups/wg-pitr.md

```diff
@@ -0,0 +1,30 @@
+# PITR Working Group
+
+PITR is an ecosystem tool for TiDB Binlog. By preprocessing the incremental backup file of TiDB, PITR merged the changes of the same line of data to produce a new, lighter incremental backup file, which greatly reduced the Time of incremental backup Recovery and realized fast-pitr (Fast Point in Time Recovery).
```

Comment:
what does `PITR` mean?

Reply:
Point in Time Recovery

Reply:
Please add this explanation to the documents.